### PR TITLE
[zh-cn]: update the translation of Window `scrollTo` method

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 1]
```diff
@@ -1,47 +1,65 @@
...
+scrollTo(x-coord, y-coord)
+scrollTo(options)
 ```
 
 ### 参数
 
-- `x-coord` 是文档中的横轴坐标。
-- `y-coord` 是文档中的纵轴坐标。
-- `options` 是一个包含三个属性的对象：
+- `x-coord` 是文档水平轴上的像素点，表示你希望在视口左上角显示的位置。
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollto/index.md:21]:
```suggestion
- `x-coord` 是你希望显示在左上角的文档水平轴像素。
```

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 2]
```diff
@@ -1,47 +1,65 @@
...
+- `y-coord` 是文档垂直轴上的像素点，表示你希望在视口左上角显示的位置。
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollto/index.md:22]:
```suggestion
- `y-coord` 是你希望显示在左上角的文档垂直轴像素。
```

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 3]
```diff
@@ -1,47 +1,65 @@
...
+
+——或者——
+
+- `options`
+  - : 一个包含以下参数的字典：
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollto/index.md:27]:
```suggestion
  - : 包含以下参数的字典：
```

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 4]
```diff
@@ -1,47 +1,65 @@
...
+    - `top`
+      - : 指定沿 Y 轴滚动窗口或元素的像素数量。
+    - `left`
+      - : 指定沿 X 轴滚动窗口或元素的像素数量。
+    - `behavior`
+      - : 确认滚动是立即完成还是平滑动画进行。此选项是一个字符串，必须取以下值之一：
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollto/index.md:33]:
```suggestion
      - : 确定滚动是即时完成还是以平滑动画进行。该选项是一个字符串，必须取以下值之一：
```

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 5]
```diff
@@ -1,47 +1,65 @@
...
+        - `smooth`: 滚动应该平滑地进行动画展示
+        - `instant`: 滚动应当瞬间完成，一次性跳跃到位
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollto/index.md:35]:
```suggestion
        - `instant`: 滚动应在一次跳转中即时完成
```

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 6]
```diff
@@ -1,47 +1,65 @@
...
+  top: 100,
+  left: 100,
+  behavior: "smooth",
 });
 ```
 
-## 注意
+## 说明
+
+{{domxref("Window.scroll()")}} 实际上与此方法相同。对于相对滚动，请参阅 {{domxref("Window.scrollBy()")}}、{{domxref("Window.scrollByLines()")}} 和 {{domxref("Window.scrollByPages()")}}。
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollto/index.md:60]:
```suggestion
{{domxref("Window.scroll()")}} 实际上与此方法相同。有关相对滚动，请参阅 {{domxref("Window.scrollBy()")}}、{{domxref("Window.scrollByLines()")}} 和 {{domxref("Window.scrollByPages()")}}。
```

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 7]
```diff
@@ -1,47 +1,65 @@
...
 
-该函数实际上和 [window.scroll](/zh-CN/docs/DOM/Window.scroll)是一样的。相对滚动可以参考 [window.scrollBy](/zh-CN/docs/DOM/Window.scrollBy)，[window.scrollByLines](/zh-CN/docs/DOM/Window.scrollByLines)，和 [window.scrollByPages](/zh-CN/docs/DOM/Window.scrollByPages)。
+对于滚动元素，请参阅 {{domxref("Element.scrollTop")}} 和 {{domxref("Element.scrollLeft")}}。
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollto/index.md:62]:
```suggestion
有关滚动元素，请参阅 {{domxref("Element.scrollTop")}} 和 {{domxref("Element.scrollLeft")}}。
```

**Review by T34-active — COMMENTED** [review]
fix content

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 8]
```diff
@@ -1,47 +1,65 @@
...
+  - : 包含以下参数的字典：
+    - `top`
+      - : 指定沿 Y 轴滚动窗口或元素的像素数量。
+    - `left`
+      - : 指定沿 X 轴滚动窗口或元素的像素数量。
+    - `behavior`
+      - : 确定滚动是即时完成还是以平滑动画进行。该选项是一个字符串，必须取以下值之一：
+        - `smooth`: 滚动应该平滑地进行动画展示
+        - `instant`: 滚动应在一次跳转中即时完成
+        - `auto`: 滚动行为由 {{cssxref("scroll-behavior")}} 的计算值来决定
```
Comment (yin1999) [inline files/zh-cn/web/api/window/scrollto/index.md:36]:
```suggestion
        - `smooth`：滚动应该平滑地进行动画展示
        - `instant`：滚动应在一次跳转中即时完成
        - `auto`：滚动行为由 {{cssxref("scroll-behavior")}} 的计算值来决定
```

File: files/zh-cn/web/api/window/scrollto/index.md [hunk 9]
```diff
@@ -1,47 +1,65 @@
...
-    top: 1000,
-    behavior: "smooth"
+  top: 100,
+  left: 100,
+  behavior: "smooth",
 });
 ```
 
-## 注意
+## 说明
```
Comment (yin1999) [inline files/zh-cn/web/api/window/scrollto/index.md:58]:
```suggestion
## 备注
```

**Review by yin1999 — COMMENTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]
Thanks @T34-active

**Pull request merged**